<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Application de gestion de rendez-vous</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>701</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="staffListWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>701</width>
       <height>310</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Personnel de soin</string>
     </attribute>
     <widget class="QTreeView" name="resourcesTreeView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>681</width>
        <height>291</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="patientSearchWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>701</width>
       <height>310</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Recherche de patients</string>
     </attribute>
     <widget class="QLabel" name="separatorLabel">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="patientSearchNameLabel">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="patientSearchIdLineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Numéro</string>
      </property>
     </widget>
     <widget class="QTableView" name="patientSearchTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>681</width>
        <height>221</height>
       </rect>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
     <widget class="QPushButton" name="launchPatientSearchButton">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>30</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="patientSearchFirstnameLineEdit">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Prénom</string>
      </property>
     </widget>
     <widget class="QLabel" name="patientSearchFirstnameLabel">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Prénom</string>
      </property>
     </widget>
     <widget class="QLabel" name="patientSearchDateLabel">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date de rendez-vous (saisir un interval)</string>
      </property>
     </widget>
     <widget class="QLabel" name="patientSearchIdLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Numéro</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="patientSearchNameLineEdit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="patientSearchPriorDateDateEdit">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>30</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="patientSearchLaterDateDateEdit">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>30</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="planningWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>701</width>
       <height>310</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Plannification de rendez-vous</string>
     </attribute>
     <widget class="QLabel" name="planningDateLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date à plannifier</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="planningDateEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="doPlanningButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Plannifier</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <widget class="QMenu" name="menuAjouter">
     <property name="title">
      <string>Ajouter</string>
     </property>
     <addaction name="editNewPatientAction"/>
     <addaction name="editNewResourceAction"/>
    </widget>
    <addaction name="quitAction"/>
    <addaction name="menuAjouter"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="learnAboutAction"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="editNewPatientAction"/>
   <addaction name="editNewResourceAction"/>
  </widget>
  <action name="quitAction">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="toolTip">
    <string>Quitter l'application</string>
   </property>
  </action>
  <action name="editNewPatientAction">
   <property name="text">
    <string>Nouveau patient</string>
   </property>
   <property name="toolTip">
    <string>Ajouter un nouveau patient</string>
   </property>
  </action>
  <action name="editNewResourceAction">
   <property name="text">
    <string>Nouveau personnel de soins</string>
   </property>
   <property name="toolTip">
    <string>Ajouter un nouveau personnel de soins</string>
   </property>
  </action>
  <action name="learnAboutAction">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>patientSearchIdLineEdit</tabstop>
  <tabstop>patientSearchNameLineEdit</tabstop>
  <tabstop>patientSearchFirstnameLineEdit</tabstop>
  <tabstop>patientSearchPriorDateDateEdit</tabstop>
  <tabstop>patientSearchLaterDateDateEdit</tabstop>
  <tabstop>launchPatientSearchButton</tabstop>
  <tabstop>patientSearchTableView</tabstop>
  <tabstop>planningDateEdit</tabstop>
  <tabstop>doPlanningButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
